bindings/virtio/virtio_net.c

```diff
@@ -224,40 +243,48 @@ void virtio_config_network(struct pci_config_info *pci)
     outb(pci->base + VIRTIO_PCI_STATUS, VIRTIO_PCI_STATUS_DRIVER_OK);
 }
 
-/* Returns 1 if there is a pending used descriptor for us to read. */
-int virtio_net_pkt_poll(void)
+/* Returns 1 if there is a pending used descriptors for us to read.
+FIXME: palainp, the description does not match the behaviour of the function */
```

Comment:
would you mind to solve this fixme? :)

Reply:
Thank you, applied in [0f92621](https://github.com/Solo5/solo5/pull/581/commits/0f926216d4f428ce2b5e086ee9d877f462979dc6).